### DISCUSSION TITLE
match_type shows in examples and json schema but not in Respec html docs?

### DISCUSSION THREAD

**Discussion by thadguidry** [reconciliation-api/specs#158] [General]
I'm a bit confused, as I look more closely at fixing various formatting issues with Respec and the defined terms we have for the API documentation.  On closer inspection, it seems that we are missing documentation on some fields?  Was that the intention where we only supply limited documentation about some fields, but that all the available fields are fully documented in the JSON schema files?  I personally would have thought that we document all the fields also in the Respec HTML documentation?  Or am I not looking at things correctly?

As an example:  `match_type` and `conditions` can be seen in examples and JSON schema files, but it's not documented in the Respec HTML.  Were we trying to have only 1 place to maintain the actual API fully and that 1 place is the JSON schema files? Confused.

**Comment by wetneb** [comment]
Yes, any differences between the JSON schemas and the spec in HTML would be worth resolving.

**Reply by thadguidry** [reply]
Resolving...meaning, both file types (index.html and .json files) should have the same field descriptions?  I.E. we are maintaining full documentation of all fields in both sets of files?  Actually, I guess full docs are the HTML and Schema (light descriptions) being in the JSON files?  Yeah?
